entrypoint.sh

```diff
@@ -13,26 +13,35 @@ function bump_version(){
 }
 
 function create_release_branch() {
-    read -p 'Bump type (possible values: patch or minor): ' release_type
+    release_type=$1
+
```

Comment:
you can add a check on the argument number then

Reply:
Well, the check is already done, I think.

There are two arguments for this function.

One mandatory for the bump type, it is checked here: https://github.com/kili-technology/kili-python-sdk/blob/2c8abb4d315cb629eeb9494a7b8798d06e2ad3f4/entrypoint.sh#L18-L21  If the bump type argument is not provided and is not equal to `minor` or `patch`, it will exit anyway.

And one optional argument for the commit hash, that will be set to HEAD by default: https://github.com/kili-technology/kili-python-sdk/blob/2c8abb4d315cb629eeb9494a7b8798d06e2ad3f4/entrypoint.sh#L25

I think however I should add some comments in the script about how to use it.